includes/Discovery.php

```diff
@@ -101,23 +101,26 @@ public function register(): void {
 	/**
 	 * Prints document title for stories.
 	 *
-	 * Works both for classic themes and block themes without any conditionals.
+	 * Works both for classic themes and block themes.
 	 *
 	 * @since 1.25.0
 	 *
 	 * @link https://github.com/GoogleForCreators/web-stories-wp/issues/12139
+	 * @link https://github.com/GoogleForCreators/web-stories-wp/issues/12487
 	 * @see _wp_render_title_tag()
 	 * @see _block_template_render_title_tag()
 	 */
 	public function print_document_title(): void {
+		$enable_metadata = function_exists( 'wp_is_block_theme' ) && wp_is_block_theme();
```

Comment:
How does this work with [Title Tag](https://codex.wordpress.org/Title_Tag).

Reply:
Since we're not relying on theme support and don't use built-in functions like `_wp_render_title_tag` or `_block_template_render_title_tag`, this shouldn't really affect us at all.

The problem with block themes is just that plugins like Yoast SEO behave differently for them, which is why we're having these issues.

You can verify this by testing a theme like Twenty Ten (does not add `title-tag` support) and Twenty Fifteen (_does_ add `title-tag` support)